packages/workorder-angular/lib/workorder-form/workorder-form-controller.js

```diff
@@ -3,44 +3,37 @@ var CONSTANTS = require('../constants');
 /**
  *
  * Controller for editing and creating workorders.
- *
- * @param $scope
- * @param mediator
- * @param workorderMediatorService
- * @param $stateParams
- * @param $q
- *
  * @constructor
  */
-function WorkorderFormController($scope, mediator, workorderMediatorService, $stateParams, $q) {
+function WorkorderFormController($scope, $state, workorderApiService, workorderFlowService, $stateParams, $q) {
```

Comment:
:+1: replacing angular's $q service with another one isn't worth it, specially since some of angular's own services use it